docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -96,7 +114,7 @@ These ports are required:
 
 | Port | Description |
 | - | - |
-| `3023` | SSH port for clients connect to. |
+| <nobr>`3023`</nobr> | SSH port for clients connect to. |
 | `3024` | SSH port used to create reverse SSH tunnels from behind-firewall environments. |
 | `443` | HTTPS connections to authenticate `tsh` users into the cluster. The same connection is used to serve a Web UI. |
```

Comment:
Do we specifically tell the user to configure the Teleport proxy to listen on port 443 in this guide?

Would it make more sense to list both the frontend and backend ports here, and have the LB direct incoming traffic on port 443 to port 3080 on the proxy? (Teleport would require additional privileges to bind to 443 as opposed to 3080)

Reply:
Looks like there's no `web_listen_addr` example here. I'll change the LB instructions!